src/test/java/org/zalando/riptide/AnyDispatchTest.java

```diff
@@ -52,8 +52,8 @@
     public AnyDispatchTest() {
         final RestTemplate template = new RestTemplate();
         template.setErrorHandler(new PassThroughResponseErrorHandler());
-        this.unit = Rest.create(template);
         this.server = MockRestServiceServer.createServer(template);
+        this.unit = Rest.create(template);
```

Comment:
Do you want to change the signature to expect `AsyncClientHttpRequestFactory` and `List<HttpMessageConverter<?>>`?